jobs/src/main/scala/dcos/metronome/utils/glue/MarathonImplicits.scala

```diff
@@ -39,18 +39,28 @@ object MarathonImplicits {
   }
 
   implicit class ConstraintSpecToProto(val spec: ConstraintSpec) extends AnyVal {
-    def toProto: marathon.Protos.Constraint = {
-      val marathonOperator = spec.operator match {
-        case Operator.Eq     => marathon.Protos.Constraint.Operator.CLUSTER
-        case Operator.Like   => marathon.Protos.Constraint.Operator.LIKE
-        case Operator.Unlike => marathon.Protos.Constraint.Operator.UNLIKE
-      }
+    def toProto: Option[marathon.Protos.Constraint] = {
+      /**
+        * Unfortunately, Metronome has always allowed valueless constraint operators, but they never had any effect.
+        *
+        * The Eq operator was replaced with Is in of favor consistency. Eq mapped to CLUSTER, and valueless cluster in
```

Comment:
strongly with parsing "with Is in of favor consistency" .  I think you mean "in favor of consistency"

Reply:
Hahahah, good catch. I can haz grammar.